build.gradle

```diff
@@ -5,8 +5,8 @@ buildscript {
         sample_app_version = '1.0'
         sample_app_code_version = 1
 
-        omise_sdk_version = '4.2.0'
-        omise_sdk_code_version = 29
+        omise_sdk_version = '4.2.1'
```

Comment:
this looks more like a feature than a patch. So if we follow https://semver.org/ should it be `4.3.0` instead?

Reply:
Good point here updated https://github.com/omise/omise-android/pull/230/commits/9050e751262f2699399f9dca520b8b1d58c9029a 🙏